graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -212,20 +108,20 @@ def _get_query_interval_of_binary_integer_inequality_filter(parameter_values, fi
             u"binary integer inequality operator {}.".format(filter_operator)
         )
 
-    query_interval = _create_integer_interval(lower_bound, upper_bound)
```

Comment:
I think that makes the code more obscure. For n-ary filter, we check that the collection has n-1 values. No reason to special case n=1, as there's also ternary filters.

Reply:
The function is called `_get_query_interval_of_binary_integer_inequality_filter` though — supplying a ternary (or any non-binary) filter to it would seem like a bug to me. I'm not sure I understand why the code would be made more obscure with a one-liner way to get the only element from the collection, care to elaborate?